File: DSA/Gautam_Shyamnani_590013078/Day1Question3.cpp
#include<iostream>
#include<vector>
using namespace std;
int main(){
    vector<int>nums={3,5,9,1,4,11};
    int n=nums.size();
    int sum=0;
    for(int i=0;i<n;i++){
        sum+=nums[i];
    }
    cout<<"The sum of all the elements of the array is: "<<sum<<endl;
    return 0;
}

File: DSA/Sarthak_Jain_590014936/DAY_1/Day_1_Question_3.py
#Sarthak Jain 590014936
#Day 1 Question 3
#Sum of Array Elements

lis = []

n = int(input("Enter number of elements:"))
for i in range(n):
    b = int(input("Enter element:"))
    lis.append(b)

print("Given List:")
print(lis)

total = 0
for x in lis:
    total += x

print("Sum of all elements in the list:", total)
